### PR TITLE
network: bridge: use device names for interfaces

### PR DESCRIPTION
**Pull request by phoracek**

### CODE REVIEW

**Comment by phoracek** [comment]
Tested on my setup, works ok.

File: pkg/network/bridge/manager.go [hunk 1]
```diff
@@ -5,12 +5,14 @@ import (
...
 	"strings"
 
+	"github.com/golang/glog"
 	"kubevirt.io/kubernetes-device-plugins/pkg/dpm"
 )
 
 const (
 	BridgesListEnvironmentVariable = "BRIDGES"
 	nicsPoolSize                   = 100
+	maxBridgeNameLength            = 12
```
Comment (mpolednik) [inline pkg/network/bridge/manager.go]: Arbitrary or some kind of system limit?

File: pkg/network/bridge/manager.go [hunk 2]
```diff
@@ -22,8 +24,11 @@ func (bl BridgeLister) Discover() *dpm.DeviceMap {
 	var devices = make(dpm.DeviceMap)
 
 	for _, bridgeName := range bridges {
+		if len(bridgeName) > maxBridgeNameLength {
+			glog.Exitf("Bridge name (%s) cannot be longer than %d characters", bridgeName, maxBridgeNameLength)
```
Comment (mpolednik) [inline pkg/network/bridge/manager.go]: Fatalf could be a better fit.

**Review by mpolednik — CHANGES_REQUESTED** [review]

**Review by mpolednik — APPROVED** [review]
LGTM codewise, could you verify that it's working at least locally?

**Comment by phoracek** [comment]
It works

```
F0222 14:34:22.220575   30603 manager.go:29] Bridge name (br10000000000000) cannot be longer than 12 characters
```

**Pull request merged**